test/services/return-requirements/purpose.service.test.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
+
+// Things we need to stub
+const FetchPurposesService = require('../../../app/services/return-requirements/fetch-purposes.service.js')
+
+// Thing under test
+const SelectPurposeService = require('../../../app/services/return-requirements/purpose.service.js')
```

Comment:
The module exported on` purpose.service.js` file is called PurposeService but here its is called SelectPurposeService. I would try and keep the naming consistent.